src/rich-text-editor.component.js

```diff
@@ -12,7 +12,7 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   const selectionRangeBeforeBlurRef = useRef(null)
   const {isFocused, setFocused} = useSynchronousFocusState()
   const bandicootId = useRef(globalBandicootId++)
-  const [lastSavedHTML, setLastSavedHTML] = useState(richTextContext.sanitizeHTML(props.initialHTML, 'setLastSavedHTML'))
+  const [lastSavedHTML, setLastSavedHTML] = useState(() => richTextContext.sanitizeHTML(props.initialHTML, 'initialSetLastSavedHTML'))
```

Comment:
useState expects a function in this form.  Otherwise it appears to get called lots of times as opposed to only on the initial renders.